### DISCUSSION TITLE
Lesson : 9 MakeFile is not working

### DISCUSSION THREAD

**Discussion by today988** [Cyfrin/foundry-full-course-cu#1589] [Q&A]
When I run,

make build

Command 'make' not found, but can be installed with:
sudo apt install make        # version 4.3-4.1build1, or
sudo apt install make-guile  # version 4.3-4.1build1

and when i install this 
[sudo] password 

what is sudo password

**Comment by EngrPips** [comment]
@today988 , i would assume it is asking for your password so as to proceed with the installation

**Reply by YavorJJ** [reply]
well same here, the problem is I forgot my password, would should I do now?

thanks, for the reply! 🫡

**Reply by EngrPips** [reply]
Maybe find a way to reset it @YavorJJ.

**Reply by kinjun751** [reply]
i have same issue i forgot my password. how do i reset it

**Reply by EngrPips** [reply]
I am unsure but maybe you can browse Google and see if there are some post that explains how to do that.

**Reply by YavorJJ** [reply]
> i have same issue i forgot my password. how do i reset it

https://www.youtube.com/watch?v=Bsl4UAfHAvs

here you are brother, I think this is going to help you, just again be sure watch the whole video or search for others if this does not help. First watch the video, then decide if this is going to work for you, be careful.

**Reply by EngrPips** [reply]
> > i have same issue i forgot my password. how do i reset it
> 
> https://www.youtube.com/watch?v=Bsl4UAfHAvs
> 
> here you are brother, I think this is going to help you, just again be sure watch the whole video or search for others if this does not help. First watch the video, then decide if this is going to work for you, be careful.

Hey @YavorJJ, It helped you reset your password, right?

**Reply by YavorJJ** [reply]
> > > i have same issue i forgot my password. how do i reset it
> > 
> > 
> > https://www.youtube.com/watch?v=Bsl4UAfHAvs
> > here you are brother, I think this is going to help you, just again be sure watch the whole video or search for others if this does not help. First watch the video, then decide if this is going to work for you, be careful.
> 
> Hey @YavorJJ, It helped you reset your password, right?

I think it was this video, I am saying to be careful, because most videos are not working or for different systems, so its better for him to watch what I sent and see if it is going to help him.

**Reply by EngrPips** [reply]
Alright.

**Reply by kinjun751** [reply]
thanks.

**Reply by EngrPips** [reply]
Anytime.

**Reply by SashaFlores** [reply]
I found an easier way to install Makefile without the need for any password or linux, all you need is powerShell and bash, follow the video below till installing GCC compiler (Don't install) after you do, install MakeFile extension on VSCode from Microsoft and allow it to compile with C++.

Very import note: after you install Scoop, shut powerShell and after you install Makefile in separate Bash, you have to close your vscode and reopen it with a new terminal for changes to take affect and then run make --version

Here is the tutorial: https://www.youtube.com/watch?v=j5gJJe_0zP4

**Reply by EngrPips** [reply]
Interesting, Thanks for this heads up @SashaFlores.

**Reply by SashaFlores** [reply]
@EngrPips Let me know if you need further help and faced any challenges, I can gladly help you out!

**Reply by EngrPips** [reply]
Yeah sure, I will reach out if I need help.
